internal/service/dns_config/api_view_resource_test.go

```diff
@@ -0,0 +1,1955 @@
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewDnssecTrustAnchors(name, "8", "tf-infoblox.com.", "false"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.algorithm", "8"),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.zone", "tf-infoblox.com."),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.sep", "false"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccViewDnssecTrustAnchors(name, "7", "tf-infoblox.com.", "true"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.algorithm", "7"),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.zone", "tf-infoblox.com."),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_trust_anchors.0.sep", "true"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccViewResource_DnssecValidateExpiry(t *testing.T) {
+	var resourceName = "bloxone_dns_view.test_dnssec_validate_expiry"
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewDnssecValidateExpiry(name, "true"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_validate_expiry", "true"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccViewDnssecValidateExpiry(name, "false"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dnssec_validate_expiry", "false"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccViewResource_DtcConfig(t *testing.T) {
+	var resourceName = "bloxone_dns_view.test_dtc_config"
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewDtcConfig(name, 300),
```

Comment:
when creating tests, try to avoid default values. Same comment applies to a lot of other tests in this file.

Reply:
In all tests containing non - boolean values , default values are replaced .